src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -138,10 +149,23 @@ const getWebhookSecret = async (uuid?: string): Promise<{ webhookSecret: string,
 			throw new Error(`Installation not found for gitHubApp with uuid ${uuid}`);
 		}
 		const webhookSecret = await gitHubServerApp.getDecryptedWebhookSecret(installation.jiraHost);
-		return { webhookSecret, gitHubServerApp };
+		/**
+		 * If we ever need to rotate the webhook secrets for Enterprise Customers,
+		 * we can add it in the array: ` [ webhookSecret ]`
+		 */
+		return { webhookSecrets: [ webhookSecret ], gitHubServerApp };
```

Comment:
How to add multiple secrets for enterprise customers? Right now `webhookSecret` DB column is of type varchar.

Reply:
We do not have to worry about the enterprise customers for now cause they have their own webhook secret. Using this as an array here to match the types. 
If we ever need to update webhook secret for any enterprise customer, we can store the webhook secret as an array similar to how we've stored in vault. So have something like "['newSecret','oldSecret']", which can still be saved in the DB. Might need to change this to parse it, but that can be done later.

Reply:
Make sense. I think this whole thing needs to be put behind FF

Reply:
When we change the webhook secret in Github's app settings, our previous code won't work in that case. So didn't add any FF.